### ISSUE TITLE
Fonctionnement de la couverture temporelle dans le parcours

### ISSUE THREAD

**Issue by agarrone** [datagouv/data.gouv.fr#1268]
Permettre de pouvoir choisir une date de début sans date de fin. 
A enquêter niveau back

![Image](https://github.com/etalab/data.gouv.fr/assets/60264344/5eaf14f4-6298-424d-ab17-6e6c4034e68e)

**Comment by maudetes** [comment]
Yes justement, ça semblerait pertinent de rendre possible ce support dans le back.
C'est aussi un cas qu'on retrouve dans le moissonnage, où aujourd'hui on attend début & fin alors que [DCAT-AP autorise](https://semiceu.github.io/DCAT-AP/releases/3.0.0/#Periodoftime) d'avoir uniquement début (ou fin par ailleurs).
Voir [cet exemple de fiche](https://data.economie.gouv.fr/explore/dataset/signalconso/information/) et [export DCAT](https://data.economie.gouv.fr/api/v2/catalog/exports/dcat?lang=fr&where=dataset_id:%27signalconso%27) côté ODS.
